go.mod

```diff
@@ -4,18 +4,18 @@ go 1.16
 
 require (
 	github.com/fluxcd/helm-controller/api v0.10.1
-	github.com/fluxcd/kustomize-controller/api v0.12.0
+	github.com/fluxcd/kustomize-controller/api v0.12.1
 	github.com/fluxcd/source-controller/api v0.13.1
 	github.com/google/uuid v1.2.0
-	github.com/jenkins-x/go-scm v1.9.0
+	github.com/jenkins-x/go-scm v1.9.1
 	github.com/olekukonko/tablewriter v0.0.0-20210304033056-74c60be0ef68
-	github.com/onsi/ginkgo v1.16.2
-	github.com/onsi/gomega v1.12.0
+	github.com/onsi/ginkgo v1.16.3
+	github.com/onsi/gomega v1.13.0
 	github.com/urfave/cli/v2 v2.3.0
-	github.com/weaveworks/profiles v0.0.2
+	github.com/weaveworks/profiles v0.0.3
 	k8s.io/api v0.20.5
 	k8s.io/apiextensions-apiserver v0.20.5
-	k8s.io/apimachinery v0.20.5
+	k8s.io/apimachinery v0.21.1
```

Comment:
this seems to be okay... was scared it would break stuff

Reply:
yeh it broke last week because there was some client-go interdependency mis-match with another thing which seems to have been resolved now